docs/JSONDocumentation.md

```diff
@@ -0,0 +1,379 @@
+
+# JSON documentation
+
+This document contains an overview of the possible keys in the intermediary JSON file. It also contains a brief explanation of the meaning of the key in the different programming languages the analyser accepts. Unless indicated otherwise, the terminology used is based on DOTNET.
+
+## Member
+This is the top level element of the file.
+
+NET
+: A member can represent a class, interface, struct or enum. By default, it is a class (if Type is not specified).
+
+Java
+: A member can represent a class, interface, record or enum. By default, it is a class (if Type is not specified).
+
+
+### Required keys:
+
+| Key      | Type   | .NET                                    | Java                                    |
+|----------|--------|-----------------------------------------|-----------------------------------------|
+| FullName | string | Name of the class/interface/struct/enum | Name of the class/interface/record/enum |
```

Comment:
Perhaps neat to document why this is "full" - I think, because it should include the namespace (C#) or package name (Java).